src/main/java/org/opensearchmetrics/metrics/release/ReleaseInputs.java

```diff
@@ -18,6 +18,7 @@ public enum ReleaseInputs {
     VERSION_2_17_0("2.17.0", "closed", "2.17", true),
     VERSION_2_18_0("2.18.0", "closed", "2.18", true),
     VERSION_2_19_0("2.19.0", "closed", "2.19", true),
+    VERSION_2_19_1("2.19.1", "open", "2.x", true),
```

Comment:
The branch should be 2.19 right ? @peterzhuamazon

Reply:
2.19